docs/docs/dcap.md

```diff
@@ -19,19 +9,33 @@ automated for you.
 As cloud providers run caches, DCAP supports vendor-specific plugins. Intel provides a default one
 which requires a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification).
 
-Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. Conclave
-bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
+Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. 
+Conclave bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can 
+be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
```

Comment:
Here I feel there should be a `,` before `as Conclave`

Reply:
Instead, I have split the sentence into two for clarity.